lib/tasks/dev.rake

```diff
@@ -0,0 +1,77 @@
+
+task sample_data: :environment do
+  p "creating sample data"
+  starting = Time.now
+
+  Member.delete_all
+  Contact.delete_all
+  Group.delete_all
+  User.delete_all
+
+  5.times do
+    number = rand.to_s[2..11] 
```

Comment:
Faker also has a [phone number](https://github.com/faker-ruby/faker/blob/master/doc/default/phone_number.md) option you can use instead of manually creating phone numbers.